doc/source/jupyterhub/customizing/user-management.md

```diff
@@ -17,12 +17,12 @@ will be lost. This is known as _culling_.
 
 ```{note}
 In JupyterHub, "inactivity" is defined as no response from the user's
-browser. JupyterHub constantly pings the user's JupyterHub browser session
+browser. JupyterHub constantly pings the user's JupyterHub browser session via the JupyterHub API
```

Comment:
Hmmm, I think there is something not right with this sentence.

JupyterHub has a REST API. `jupyterhub-idle-culler` could use that REST API, and external user pods could use it - but it doesn't make sense for JupyterHub itself to use it right?

Reply:
Yes- you're right that is not clear. I'll take another pass.